docs/code.rst

```diff
@@ -381,6 +382,7 @@ Community contributed hooks
 .. autoclass:: airflow.contrib.hooks.vertica_hook.VerticaHook
 .. autoclass:: airflow.contrib.hooks.spark_jdbc_hook.SparkJDBCHook
```

Comment:
I know this is not your entry... but it looks like this is not in order either.

Reply:
Ad a comment for future developers to maintain the alphabetic ordering of these entries.

Reply:
fixed, commit coming